src/utils/vue.ts

```diff
@@ -0,0 +1,50 @@
+import { TextDocument, Position } from 'vscode-languageserver';
+
+export function isVueFile(path: string) {
+	return path.endsWith('.vue');
+}
+
+export function getVueSCSSRegions(content: string) {
+	const regions: Array<[number, number]> = [];
```

Comment:
If it's not too much trouble, extract the `Array<[number, number]>` type to the type alias, because this type is used more than once.

Reply:
Good suggestion